docker-build/nginx.conf

```diff
@@ -0,0 +1,15 @@
+upstream config {
+    server wsgi:8000;
+}
+
+server {
+
+    listen 80;
+
+    location / {
```

Comment:
we have different platforms for a single domain getkeel.com
cant put on location / to only django app